src/_pytest/mark/__init__.py

```diff
@@ -181,7 +183,9 @@ def from_item(cls, item: "Item") -> "KeywordMatcher":
 
         return cls(mapped_names)
 
-    def __call__(self, subname: str) -> bool:
+    def __call__(self, subname: str, /, **kwargs: object) -> bool:
```

Comment:
`kwargs` is always a mapping, I don't think that using `object` for typing such things is customary, is it?

Reply:
Switched to `str | int | bool | None` for now.